graphbolt/src/csc_sampling_graph.cc

```diff
@@ -207,6 +200,34 @@ c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::SampleNeighbors(
       subgraph_reverse_edge_ids, subgraph_type_per_edge);
 }
 
+c10::intrusive_ptr<SampledSubgraph> CSCSamplingGraph::SampleNeighbors(
+    const torch::Tensor& nodes, const std::vector<int64_t>& fanouts,
+    bool replace, bool labor, bool return_eids,
```

Comment:
I still see the mix-usage of layer (in header file) and labor (here), since the apis in python are clarified, maybe change them all to labor or layer or layer_neighbor? either should be fine now.

Reply:
We can make this argument layer since it enables layer sampling functionality for neighbor. But the underlying algorithm powering this is labor, so we can keep the implementation details labor if you would like.